packages/contracts-bedrock/src/dispute/lib/LibGameArgs.sol

```diff
@@ -0,0 +1,41 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.15;
+
+/// @title LibGameArgs
+/// @notice Library for decoding the game arguments used in dispute games.
+library LibGameArgs {
+    /// @notice Decodes the game arguments from a bytes array.
+    /// @param _gameArgs The bytes array containing the encoded game arguments.
+    function decode(bytes memory _gameArgs)
```

Comment:
Nice!  What do you think about returning a struct of decoded values to help avoid stack-too-deep errors?